src/lib/support/DefaultStorageKeyAllocator.h

```diff
@@ -129,6 +129,15 @@ class DefaultStorageKeyAllocator
     }
 
     static StorageKeyName AccessControlExtensionEntry(FabricIndex fabric) { return StorageKeyName::Formatted("f/%x/ac/1", fabric); }
+    static StorageKeyName AccessControlCommissioningArlEntry(size_t index)
```

Comment:
can this be wrapped in the ifdef?

if CHIP_CONFIG_USE_ACCESS_RESTRICTIONS

Reply:
done

Reply:
i hit a build issue when including the header for the build flag, so I left it in for now.